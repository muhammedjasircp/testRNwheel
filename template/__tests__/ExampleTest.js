import 'react-native';
import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import Example from '../src/screens/ExampleScreen';

test('renders correctly', () => {
  const tree = renderer.create(<Example />).toJSON();
  expect(tree).toMatchSnapshot();
});
