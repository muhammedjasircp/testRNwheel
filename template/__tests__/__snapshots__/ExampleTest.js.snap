// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ccc",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hello
  </Text>
</View>
`;
